packages/mgt-chat/src/statefulClient/StatefulGraphChatClient.ts

```diff
@@ -396,17 +392,14 @@ class StatefulGraphChatClient implements StatefulClient<GraphChatClient> {
         // TODO: in parallel promise resolving, find out how to trigger different
         // TODO: state for failed subscriptions in GraphChatClient.onSubscribeFailed
         await this.loadChatData();
-        await this._notificationClient.subscribeToChatNotifications(
-          this._userId,
-          this._chatId,
-          this._eventEmitter,
-          () =>
-            this.notifyStateChange((draft: GraphChatClient) => {
-              draft.status = 'subscribing to notifications';
-            })
+        await this._notificationClient.subscribeToChatNotifications(this.userId, this.chatId, this._eventEmitter, () =>
+          this.notifyStateChange((draft: GraphChatClient) => {
+            draft.status = 'subscribing to notifications';
+          })
         );
-      } catch (error) {
-        if (error instanceof GraphError) {
+      } catch (e) {
+        console.error(e);
```

Comment:
Perhaps this could have a leading message string to provide context as to where this error is coming from?